### PR TITLE
UI dashboard main page left panel

### PR DESCRIPTION
**Pull request by GuohaoOuyang**
Hi, check this out. Don't worry about the hard-coded listItem for now, it will be optimized later. Going to work on the title ellipsis now

### CODE REVIEW

File: client/src/layout/MentionList.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    width: 20,
+    height: 20,
+  },
+  track: {
+    borderRadius: 25 / 2,
+    border: "none",
+    backgroundColor: "#bdc7d4",
+    opacity: 1,
+  },
+  checked: {},
```
Comment (WhoaNellie) [inline client/src/layout/MentionList.js]: These can also be put in a/the same makeStyles, but you also might consider putting the CustomizedSwitch in its own file

File: client/src/pages/HomePage.js [hunk 2]
```diff
@@ -1,13 +1,73 @@
...
+}));
 
 const HomePage = () => {
+  const classes = useStyles();
+  const mentionDatas = [
+    {
+      title: "PayPal invested $400 in DolphinCorp",
+      id: 1,
+      platform: "Reddit",
+      image: "https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png",
```
Comment (WhoaNellie) [inline client/src/pages/HomePage.js]: Could you replace these with the images from utils/images?

File: client/src/layout/MentionList.js (hunk 1 above)
Comment (GuohaoOuyang) [reply client/src/layout/MentionList.js]: Yeah, CustomizedSwitch has been separated.

File: client/src/pages/HomePage.js (hunk 2 above)
Comment (GuohaoOuyang) [reply client/src/pages/HomePage.js]: It's a bit awkward the image folder has been put under src, can't be accessed through the relative path. I add some random pics

File: client/src/pages/HomePage.js [hunk 3]
```diff
@@ -1,28 +1,68 @@
...
         "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
     },
   ];
 
-  const k = match.params;
   return (
     <>
       <AppBarLoggedIn />
-      {console.log(k)}
       <Grid container style={{ backgroundColor: "#fff", height: "100vh" }}>
```
Comment (jessdesr) [inline client/src/pages/HomePage.js:56]: Preferred here to avoid styling in multiple locations: move these inline styles to a class in useStyles

**Review by jessdesr — APPROVED** [review]
Nice job on matching the look of the designs, small comment but can be changed after the merge.

File: client/src/pages/HomePage.js (hunk 3 above)
Comment (GuohaoOuyang) [reply client/src/pages/HomePage.js:56]: sure, I will move these inline styles to makestyle after.

**Pull request merged**